packages/owlbot-bootstrapper/common-container/cloudbuild-deploy-container.yaml

```diff
@@ -35,4 +35,9 @@ steps:
   - name: gcr.io/cloud-builders/docker
     id: "push-docker"
     waitFor: ["build-docker"]
-    args: ["push", "gcr.io/$PROJECT_ID/owlbot-bootstrapper", "us-docker.pkg.dev/owlbot-bootstrap-prod/owlbot-bootstrapper-images/owlbot-bootstrapper"]
\ No newline at end of file
+    args: ["push", "gcr.io/$PROJECT_ID/owlbot-bootstrapper"] 
+
+  - name: gcr.io/cloud-builders/docker
+    id: "push-docker"
```

Comment:
I think this will fail with duplicate `id` fields.

Alternatively, use the `images` key at the end of the cloudbuild yaml: https://cloud.google.com/build/docs/build-config-file-schema#images